recipes-extended/onl/files/accton-as5835-54x/0001-Support-psu_fan_dir-sysfs-for-YM-1401A-PSU.patch

```diff
@@ -44,7 +40,10 @@ index 5964e28a..3a94da50 100755
 +                else {
 +                    data->mfr_serial[ARRAY_SIZE(data->mfr_serial)-1] = '\0';
 +                }
-+
+ 
+-            if (status < 0) {
+-                dev_dbg(&client->dev, "reg %d, err %d\n", command, status);
+-                goto exit;
```

Comment:
The commit message does not cover the change in ym2651y.c. Is this change intentional?

Reply:
I assume different git version decided to produce a different diff, since the effective change looks identical (instead of deleting an empty line, then adding one again, it added stuff before it and left the empty line intact).